bgcval2/bgcval2_make_report.py

```diff
@@ -1690,6 +1635,8 @@ def newImageLocation(fn):
                 print("Adding ", relfn, "to script")
 
         if len(otherFilenames):
+            # I think this is never happens anymore.
+            assert 0
```

Comment:
```suggestion
```

Reply:
this causes the local tests to fail, I don't know what's the use for it, but I know me tests shouldn't fail bc of this :grin:

Reply:
The test may be testing something that should never happen then, lol. I'll have a look.

Reply:
cheers :beer:

Reply:
It looks like it's passing now? Which test was it that failed?

Reply:
`tests/integration/test_analysis_compare_end-end.py` - does it pass locally for you? If so, you + Github Actions are both telling me am drunk so am gonna go home :grin:

Reply:
leave it, all good test-wise (although I really don't like these antiquated assert statements - you need to plop real checks that raise real exceptions :grin: )